common/src/main/java/com/microsoft/identity/common/internal/commands/DeviceCodeFlowCommand.java

```diff
@@ -0,0 +1,64 @@
+//  Copyright (c) Microsoft Corporation.
+//  All rights reserved.
+//
+//  This code is licensed under the MIT License.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files(the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions :
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+package com.microsoft.identity.common.internal.commands;
+
+import android.content.Intent;
+
+import androidx.annotation.NonNull;
+
+import com.microsoft.identity.common.internal.commands.parameters.DeviceCodeFlowCommandParameters;
+import com.microsoft.identity.common.internal.controllers.BaseController;
+import com.microsoft.identity.common.internal.result.AcquireTokenResult;
+
+public class DeviceCodeFlowCommand extends TokenCommand {
+    /**
+     * Relevant error codes and messages go here
+     */
+
+    public static DCFCommandCallback dcfCallback;
+
+    public DeviceCodeFlowCommand(@NonNull DeviceCodeFlowCommandParameters parameters,
+                                 @NonNull BaseController controller,
+                                 @NonNull CommandCallback callback,
+                                 @NonNull String publicApiId) {
+        super(parameters, controller, callback, publicApiId);
+
+        dcfCallback = (DCFCommandCallback) callback;
```

Comment:
This should be taken care of by the parent

Reply:
I'm not sure what the better way to do this is. If I change the type of the parameter to DeviceCodeFlowCommandCallback, would that be enough?